xsnap/sources/xsnapPlatform.c

```diff
@@ -804,20 +851,20 @@ void fxVerifyEnvironment(txMachine* the, txVerifyList *list, txSlot* environment
 	txVerifyLink link;
 	txSlot* prototype;
 	txSlot* property;
-
+	
```

Comment:
Below here, these are mostly dangling whitespace changes that will thrash, for example, with any editor configuration that automatically strips dangling whitespace on save.